types/JudgeType.d.ts

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
JudgeType.ts filename should be lowercase, judgetype.ts

The type itself should have PasacalCase, JudgeType

Reply:
Ok, We will correct it.